src/main/java/org/synyx/urlaubsverwaltung/statistics/vacationoverview/web/AbsenceOverviewDayDto.java

```diff
@@ -0,0 +1,30 @@
+package org.synyx.urlaubsverwaltung.statistics.vacationoverview.web;
+
+public class AbsenceOverviewDayDto {
```

Comment:
Not sure whats better:
a) AbsenceOverviewDayDto for both, `tableHeadDay` and persons day absence
b) Two specific Types: e.g. `AbsenceOverviewDayDto` and `AbsenceOverviewPersonDayDto` (which extends AbsenceOverviewDayDto with additional field `type`.

Reply:
no inheritance 👿

I've introduced separate DTOs for the template head cells and the person cells.